nix/pkgs/default.nix

```diff
@@ -91,6 +88,15 @@ let
     inherit git-rev;
   };
 
+  # Thorp is an S3 sync tool used for deployments
+  thorp =
+    let mvn2nix = import sources.mvn2nix { };
+    in
+    pkgs.callPackage ./thorp {
+      thorpSrc = sources.thorp;
+      inherit mvn2nix;
+      inherit (pkgs) stdenv jdk11_headless maven makeWrapper graphviz;
```

Comment:
You can actually drop this line. `callPackage` fills those in for you.

(The idea behind `callPackage`is that it looks at the arguments of a function and tries to apply all matching attributes from the set that it has been called from. In this case `pkgs` can provide everything apart from `thorpSrc` and `mvn2nix` which you have to inherit specifically).